apps/ledger-live-desktop/src/newArch/WalletSync/hooks/useTrustchainSdk.ts

```diff
@@ -5,6 +5,10 @@ import { getEnv } from "@ledgerhq/live-env";
 import { getSdk } from "@ledgerhq/trustchain/index";
 import { withDevice } from "@ledgerhq/live-common/hw/deviceAccess";
 import Transport from "@ledgerhq/hw-transport";
+import { trustchainLifecycle } from "@ledgerhq/live-wallet/walletsync/index";
+import { useStore } from "react-redux";
+import { walletSelector } from "~/renderer/reducers/wallet";
+import { wsStateSelector } from "@ledgerhq/live-wallet/store";
```

Comment:
I feel like we should avoid abbreviations as much as possible for clarity reasons especially when it's in a package that isn't specific to Wallet Sync

Reply:
`wsStateSelector` because it gets the `wsState: WSState` field of the store. we could globally rename it. what would you suggest then? basically it's a object with `{ data : the latest distance state , version : the version associated to it }`

Reply:
We could at least rename those to `walletSyncState` and `walletSyncStateSelector` to understand immediately that they are used for this feature

Reply:
or maybe it's actually `cloudSyncState` 👀  except it's the one specific to `live` schema.